websites/S/StatusBot/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Dylan Davies",
+      "id": "350707981178109964"
+    },
+    "url": [
+      "beta.statusbot.xyz",
+      "statusbot.xyz",
+      "s.statusbot.xyz"
+    ],
+    "description": {
+      "en": "Website presence integration for StatusBot"
+    },
+    "service": "StatusBot",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/hZ0ONFr.png",
+    "thumbnail": "https://i.imgur.com/hZ0ONFr.png",
```

Comment:
Please change the thumbnail to a promotional image or a screenshot of the website.

Reply:
Done